packages/ts-transform/src/class-names/__tests__/index.test.tsx

```diff
@@ -332,8 +332,8 @@ describe('class names transformer', () => {
         );
       `);
 
-      expect(actual).toInclude('style={{ "--var-test-fontsize20": fontSize + "px" }}');
-      expect(actual).toInclude('.css-test{font-size:var(--var-test-fontsize20)}');
+      expect(actual).toInclude('<div style={{}} className={"css-test"}>hello, world!</div>');
```

Comment:
can you update this test to use a dnamic value? maybe throw in a hook use

```
const [fontSize] = useState(12);
```

Reply:
```js
const fontSize = true && '12px';
```

Will this work for dynamic values, for `useState` I think I'll have to add a mock in the `transformer`.